tests/fortex/health/readers/CT_image_reader_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for CTimageReader
+"""
+
+import os
+import unittest
+from ddt import ddt, data
+
+from forte.pipeline import Pipeline
+from forte.data.data_pack import DataPack
+from fortex.health.readers.CT_image_reader import CTimageReader
+import numpy as np
+import pydicom
+
+
+class CTimageReaderDirPipelineTest(unittest.TestCase):
+    def setUp(self):
+        self.orig_image_pth: str = os.path.abspath(
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                os.pardir,
+                os.pardir,
+                os.pardir,
+                os.pardir,
+                "examples/windowing_CT_Scans/sample_data/",
+            )
+        )
+
+        self.expected_image_path = os.listdir(self.orig_image_pth)
+
+        self.reader = CTimageReader()
+        self.pl = Pipeline[DataPack]()
+        self.pl.set_reader(self.reader)
+        self.pl.initialize()
+
+    def test_reader(self):
+        for pack in self.pl.process_dataset(self.orig_image_pth):
+            self.assertTrue(
+                pack.pack_name.split("/")[-1] in self.expected_image_path
```

Comment:
`pack_name` isn't the place for storing paths. If we want to store path, we have discussed to use `Payload`